tools/topology/sof-tgl-nocodec-ci.m4

```diff
@@ -24,6 +24,27 @@ include(`sof/tokens.m4')
 include(`platform/intel/tgl.m4')
 include(`platform/intel/dmic.m4')
 
+# Define pipeline id for intel-generic-dmic-kwd.m4
+# to generate dmic setting with kwd when we have dmic
+# define channel
+define(CHANNELS, `2')
+# define kfbm with volume
+define(KFBM_TYPE, `vol-kfbm')
+# define pcm, pipeline and dai id
+define(DMIC_PCM_48k_ID, `99')
+define(DMIC_PIPELINE_48k_ID, `10')
+define(DMIC_DAI_LINK_48k_ID, `6')
+define(DMIC_DAI_LINK_48k_NAME, `NoCodec-6')
```

Comment:
didn't we already say that this is a horrible name and this is not aligned with the kernel PR?
Really don't get what the directions are.

Reply:
@plbossart I think the kernel PR was abandoned and closed, so the naming is kept intact.